meson.build

```diff
@@ -269,6 +269,12 @@ if get_option('plugin_thunderbolt')
   conf.set('HAVE_THUNDERBOLT', '1')
 endif
 
+if get_option('plugin_flashrom')
+  flashrom = find_program('flashrom', required: false)
```

Comment:
The application itself isn't used anymore right?  This should be possible to drop now.